livemaps-api/APIFunctions/RoomData.cs

```diff
@@ -168,12 +164,12 @@ private static object GetLevelName(string levelId)
             }
         }
 
-        private static async Task<IEnumerable<BuildingConfig>> FetchAtlasConfig(CloudBlockBlob configRef)
+        private static async Task<IEnumerable<BuildingConfig>> FetchAtlasConfig(BlobClient configRef)
         {
             BuildingConfig[] cfg;
             using (var ms = new MemoryStream())
             {
-                await configRef.DownloadToStreamAsync(ms);
+                await configRef.DownloadToAsync(ms);
```

Comment:
Github won't let me select the entire region since most of it's outside the PR, but we should replace this entire method with something that lets BinaryData do all the work:
```C#
BlobDownloadResult result = await configRef.DownloadContentAsync();
BuildingConfig[] config = result.Content.ToObjectFromJson<BuildingConfig[]>();
return config;
```